csv_detective/detect_labels/other/floats/__init__.py

```diff
@@ -6,7 +6,7 @@
 def _is(header):
     '''Returns 1 if the (processed) header matches one of the expected words combination, else 0'''
 
-    words_combinations_list = [] #Not relevant to make it match with exact words
+    words_combinations_list = ['part', 'ratio']
```

Comment:
If this check is not fulfilled, then the label score will be 0, which means the overall column score (label + field) can only reach 0.5 in the current setting

Reply:
@sixtedemaupeou  That is not satisfying I think because there is plenty of float field columns with an infinity of possibility for column names. But maybe with your new scoring method, that does not matter.